Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM golang:1.15.2-buster as builder
```

Comment:
Since updates to this repo are infrequent we can probably do `golang:latest` here